src/components/popover-container/popover-container.d.ts

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
-
 import { AlignBinaryType } from '../../utils/helpers/options-helper/options-helper';
+import { PaddingSpacingProps } from "../../utils/helpers/options-helper";
```

Comment:
I think that could just be:

```suggestion
import OptionsHelper from "../../utils/helpers/options-helper";
```

Reply:
good point missed the file imports had the same path